index.bs

```diff
@@ -201,13 +201,13 @@ The <dfn attribute for="XRSession">environmentBlendMode</dfn> attribute MUST rep
 XR Compositor Behaviors {#xr-compositor-behaviors}
 ---------------------
 
-When presenting content to the [=XR device=], the [=XR Compositor=] MUST apply the appropriate <dfn>blend technique</dfn> to combine virtual pixels with the [=real-world environment=]. The appropriate technique is determined based on the [=XR device=]'s [=display technology=] and the [=XRSession/mode=].
```

Comment:
I think this comment makes it more confusing.
The WebXR core spec should clarify that buffers are premultiplied and the compositing spec already defines how non-premultiplied and premultiplied pixels should be handled.

Reply:
Do you think there are any changes that should be made here to address #14, then?

Reply:
Sorry, I also think that this makes things potentially more confusing. As per the issue #14 discussion, whenever a buffer with alpha channel is handed off through an API from a producer to a consumer for blending, the consumer needs to know if the (source_rgb * source_alpha) multiply was already done by the producer, or if the consumer needs to do so. There's no way to tell by looking at the pixels or buffer, this piece of information needs to be communicated through either an explicit flag, or through a well-defined convention if there is no choice.

"MAY assume" isn't sufficient, it needs to be a very specific statement.

Unfortunately I'm very short on time at the moment, could we table this until TPAC if we don't have consensus?

Reply:
Yeah, that's fair. I think all these issues are covered by the core spec issues I filed (which we will discuss at TPAC), so we can keep #14 closed for now